client/signup/config/flows.js

```diff
@@ -166,6 +167,9 @@ function removeUserStepFromFlow( flow ) {
 
 function getCurrentFlowNameFromTest() {
 	// No tests currently running
+	if ( abtest( 'freeTrials' ) === 'offered' ) {
```

Comment:
`getCurrentFlowNameFromTest` is called for every user, meaning that every user will be assigned to the `freeTrials` test. Is this intentional?